### PR TITLE
simple addition calculation page

### PR DESCRIPTION
**Pull request by Kavster92**
[DEMO LINK](https://Kavster92.github.io/react_sum/)

### CODE REVIEW

**Review by lefteris-L — APPROVED** [review]
The demo link doesn't seem to work. Maybe you didn't deploy? The code is 💯

**Review by dolnaro — APPROVED** [review]
except Demo Link GJ!

**Comment by dolnaro** [comment]
Now it's working.

File: src/components/Sum.jsx [hunk 1]
```diff
@@ -1 +1,9 @@
-// export function Sum() {}
+import React from 'react';
+
+export function Sum({ a = 0, b = 0 }) {
```
Comment (naziamoff) [inline src/components/Sum.jsx:3]: better to make your components as arrow functions

**Review by naziamoff — APPROVED** [review]

**Pull request closed without merging**